pages/glossary/programatically-determined-link-context.md

```diff
@@ -0,0 +1,20 @@
+--
+title: Programmatically Determined Link Context
+key: programmatically-determined-link-context
+--
+
+In HTML, information that is programmatically determinable from a link includes [text nodes][] that are [included in the accessibility tree][] and are
+
+- sibling [phrasing content][] nodes in the same [paragraph][] as the link;
```

Comment:
I don't think paragraph works here. I wager all screen readers look at `p` elements for paragraphs, not this HTML spec definition of it.